source/fundamentals/monitoring/cluster-monitoring.txt

```diff
@@ -85,7 +85,7 @@ The following sections show sample output for each type of SDAM event.
 serverDescriptionChanged
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: json
+.. code-block:: js
```

Comment:
A couple items here:
- I think "js" might be equivalent to "javascript" (need to check pygments reference docs), but I noticed that the old version of the docs has proper syntax highlighting and this one does not:
https://docs.mongodb.com/drivers/node/current/fundamentals/monitoring/cluster-monitoring/#serverdescriptionchanged

I think part of the goal should not be to remove syntax highlighting where it existed before unless it was incorrect. Could you check the current documentation and compare them with all the instances you've changed to make sure this goal is met?